### PR TITLE
Website update - Vuepress

### PR DESCRIPTION
**Pull request by DavertMik**
Moved website from docusaurus to vuepress

* it is faster
* we can include more links into navigation
* more flexible in styling content
* has dropdowns fro helpers!

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+        //     { text: "advanced", link: "advanced" },
+        //     { text: "hooks", link: "hooks" },
+        //   ],
+        // }
+      ],
+      // "Tutorials": [
+      // { text: //   "books, link: //   "books" },
+      // { text: //   "videos, link: //   "videos" },
+      //   "examples"
+      // ]
```
Comment (PeterNgTr) [inline docs/.vuepress/config.js]: should we remove those comment lines?

File: docs/.vuepress/theme/components/Footer.vue [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        <h4 class="important">Commercial Support</h4>
+        <ul>
+          <li><a href="http://sdclabs.com/codeceptjs?utm_source=codecept.io&utm_medium=footer&utm_term=link&utm_campaign=reference">Consulting</a></li>
+          <li><a href="http://sdclabs.com/trainings/web-automation-codeceptjs?utm_source=codecept.io&utm_medium=top_menu&utm_term=link&utm_campaign=reference">Trainings</a> </li>
+          <li><a href="https://sdclabs.com/#services">Hire Engineers</a></li>
+        </ul>
+      </div>
+    </section>
+    <div class="copyright">
+    <h5>CodeceptJS - supercharged end 2 end testing framework for NodeJS</h5>
```
Comment (PeterNgTr) [inline docs/.vuepress/theme/components/Footer.vue]: `CodeceptJS - Supercharged E2E testing framework for NodeJS`

File: docs/.vuepress/theme/components/Home.vue [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+  <Content slot-key="code"></Content>
+</div>
+
+<section class="testimonials">
+  <div class="inner">
+  <h2 class="text-center">Trusted By Enterprises</h2>
+  <div class="row">
+    <div class="quote">
+      <p>
+        We have been using CodeceptJS as our UI testing framework, and it has made writing tests so simp  le for us, the amount of options and features available in CodeceptJS just out of the box are perfect for us to <b>test an application like Percona Monitoring and Management</b> (PMM), with so many dashboards & Metric plots.
```
Comment (PeterNgTr) [inline docs/.vuepress/theme/components/Home.vue]: `simp  le` not sure why we have space in between.

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631&bid=15637124)

File: runio.js [hunk 4]
```diff
@@ -35,21 +36,76 @@ module.exports = {
...
+
+## Recipes
+
+`;
+    const res = await axios.get('https://codecept.discourse.group/search.json?q=category%3A9');
+    for (const topic of res.data.topics) {
+      if (topic.slug === 'about-the-continuous-integration-category') continue;
+      body += `* ### [${topic.title}](https://codecept.discourse.group/t/${topic.slug}/)\n`;
+    }
+    writeToFile('docs/continuous-integration.md', cfg => cfg.line(body));
```
Comment (DavertMik) [inline runio.js]: Codacy Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

File: runio.js [hunk 5]
```diff
@@ -114,78 +173,110 @@ module.exports = {
...
     // publish wiki pages to website
-    if (!fs.existsSync('website/wiki/Home.md')) {
+    if (!fs.existsSync('docs/wiki/Home.md')) {
       await git((fn) => {
-        fn.clone('git@github.com:Codeception/CodeceptJS.wiki.git', 'website/wiki');
+        fn.clone('git@github.com:Codeception/CodeceptJS.wiki.git', 'docs/wiki');
       });
     }
-    await chdir('website/wiki', () => git(cfg => cfg.pull('origin master')));
+    await chdir('docs/wiki', () => git(cfg => cfg.pull('origin master')));
```
Comment (DavertMik) [inline runio.js]: Codacy Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

**Comment by DavertMik** [comment]
I updated PR and moved out all website related things to https://github.com/codecept-js/website
So this PR actually simplifies development, as all styling and all static pages no longer require updates to main repo.

However, all docs/* still need to be formatted for vuepress and runio script was used to do that.

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631&bid=15637124)

File: runio.js (hunk 4 above)
Comment (DavertMik) [inline runio.js]: Codacy Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

File: runio.js (hunk 5 above)
Comment (DavertMik) [inline runio.js]: Codacy Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631&bid=15637124)

File: runio.js (hunk 4 above)
Comment (DavertMik) [inline runio.js]: Codacy Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

File: runio.js (hunk 5 above)
Comment (DavertMik) [inline runio.js]: Codacy Issue found: [Expected parentheses around arrow function argument.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=4672631)

**Pull request merged**